File: web_photos/pages/trip/calcTrip.py
#! /usr/bin/python3

import cgi
import sys
import os

form = cgi.FieldStorage()

priceLocation = 0
continente = form.getvalue('continente')
n_dias = form.getvalue('n_dias')

if continente == 'america':
	priceLocation = 4
elif continente == 'europa':
	priceLocation = 1
elif continente == 'asia':
	priceLocation = 3
elif continente == 'africa':
	priceLocation = 2
elif continente == 'oceania':
	priceLocation = 5
else:
	priceLocation = 0

total = int(n_dias) * int(priceLocation)

print(f'total = {str(total)}')

File: web_photos/pages/trip/calcTrip.php
<?php

parse_str(getenv("QUERY_STRING"), $_GET);

$continente = $_GET['continente'];
$n_dias = $_GET['n_dias'];

$priceLocation = 0;

if ($continente == 'america') {
    $priceLocation = 4;
} elseif ($continente == 'europa') {
    $priceLocation = 1;
} elseif ($continente == 'asia') {
    $priceLocation = 3;
} elseif ($continente == 'africa') {
    $priceLocation = 2;
} elseif ($continente == 'oceania') {
    $priceLocation = 5;
} else {
    $priceLocation = 0;
}

$total = intval($n_dias) * $priceLocation;

echo "total = " . strval($total);
?>
